### ISSUE TITLE
Gradient not loading on iOS

### ISSUE THREAD

**Issue by gustavf** [dotnet/maui#9854]
### Description

Sorry, got on more problem with the gradient or possibly the frame in iOS. On initial loading the app the background is white instead of drawing the expected coloured backgroun. Android seems not to have this issue. Other controls seem fine (label, buttons, images etc).

### Steps to Reproduce

I start with frame/gradient:

```XAML
<Frame>
    <Frame.Background>
        <LinearGradientBrush EndPoint="0,1">
            <GradientStop Color="Yellow" 
                                  Offset="1.0" />
            <GradientStop Color="Blue" 
                                  Offset="0.1" />
        </LinearGradientBrush>
    </Frame.Background>
</Frame>
```

When I load this on iOS simulator and device, it shows a white screen (other controls like labels are showing fine). If I bring it to the background, look at the home screen and then bring it back, then it draws my background (my gradient in this case). 

### Link to public reproduction project repository

n/a

### Version with bug

6.0.312

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

latest iOS (15.5), both simulator and device

**Comment by gustavf** [comment]
Repo available here https://github.com/gustavf/mauibug
Also added two screenshots. It's enough to lock/unlock the screen and the gradient will show correctly.

**Comment by jfversluis** [comment]
Might be fixed by #7925
